styles/cobalt/LICENSE.md

```diff
@@ -0,0 +1,3 @@
+The files in this directory were created by [GitLab](https://about.gitlab.com/), licensed under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/). We modified the contents of the files for Cobalt styles.
+
+For the current versions of these files, see https://gitlab.com/gitlab-org/gitlab/-/tree/master/doc/.vale/gitlab.
```

Comment:
@mjang-cobalt I added a new LICENSE file instead. Thank you for your recommendations.

Reply:
Thanks for your patience, @ana-dashuk-cobalt .